packages/angular_devkit/build_angular/test/browser/differential_loading_spec_large.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Architect } from '@angular-devkit/architect/src/index2';
+import { join, normalize } from '@angular-devkit/core';
+import { updateJsonFile } from '../../../../../tests/legacy-cli/e2e/utils/project';
+import { browserBuild, createArchitect, host } from '../utils';
+
+
+describe('Browser Builder differential loading', () => {
+  const target = { project: 'app', target: 'build' };
+  let architect: Architect;
+
+  beforeEach(async () => {
+    await host.initialize().toPromise();
+    architect = (await createArchitect(host.root())).architect;
+  });
+
+  // Temporarily commented out
+  // afterEach(async () => host.restore().toPromise());
+
+  it('works', async () => {
+
+    const tsConfigJsonPath = join(host.root(), normalize('tsconfig.json'));
+
+    // HACK: Somehow there is currently an issue under Windows
+    const tsConfigJsonPathCorrected = tsConfigJsonPath.replace('/C/', 'C:/');
```

Comment:
@alan-agius4 This is what i meant

Reply:
Let’s take it in slack, and I’ll explain what you need to do.

Reply:
Thx! This helped!